src/Http/Http.Results/src/Results.cs

```diff
@@ -588,5 +588,11 @@ public static IResult Accepted(string? uri = null, object? value = null)
         /// <returns>The created <see cref="IResult"/> for the response.</returns>
         public static IResult AcceptedAtRoute(string? routeName = null, object? routeValues = null, object? value = null)
             => new AcceptedAtRouteResult(routeName, routeValues, value);
+
+        /// <summary>
+        /// Provides a place for external libraries to extend the default `Results` set
```

Comment:
```suggestion
        /// Provides a place for external libraries to extend the default <see cref="Results"/> set
```

Does this not work?

Reply:
It does. Apparently, GH only lets you pick one suggestion against a single diff so I picked Damian's. I thought it had this change but looks like it didn't.